scripts/main.js

```diff
@@ -1,5 +1,40 @@
 var helloCork = function() {
-  console.log('Hello Cork');
+  	console.log('Hello Cork');
 };
 
+var signedUrl = "https://api.meetup.com/cork-javascript-meetup/events?desc=true&photo-host=public&page=20&sig_id=210143296&status=upcoming%2Cpast&sig=0607ea8f24a049a251ec629255d393a89c8ece7f";
+var call;
+
+var getMeetups = function() {
+	call = $.ajax({
+		type: "GET",
+		dataType: "jsonp",
+		url: signedUrl,
+
+		success: updateDOM
+	});
+	//console.log(call);
+}
+
+var updateDOM = function() {
+	var events = (call.responseJSON.data);
```

Comment:
I agree. Just had no idea I could do this.

So my ajax call returns all the objects as public variables?